qdrant_client/conversions/conversion.py

```diff
@@ -1255,7 +1255,7 @@ def convert_sparse_index_config(cls, model: grpc.SparseIndexConfig) -> rest.Spar
             full_scan_threshold=model.full_scan_threshold
             if model.HasField("full_scan_threshold")
             else None,
-            on_disk=model.on_disk if model.HasField("on_disk") else None,
+            index_type=model.on_disk if model.HasField("on_disk") else None, # TODO the types are incompatible for a conversion?
```

Comment:
I have a problem here, the types do not align.
On one side an Optional<bool> and on the other side an enum.